api/include/opentelemetry/logs/logger.h

```diff
@@ -55,25 +98,43 @@ class Logger
   virtual void EmitLogRecord(nostd::unique_ptr<LogRecord> &&log_record) noexcept = 0;
 
   /**
-   * Emit a Log Record object with custom span context
+   * Emit a Log Record object with arguments
    *
-   * @param log_record
+   * @param log_record Log record
+   * @tparam args Arguments
```

Comment:
Nit - should we mention the allowed parameters (in any order), as this will go into the API documentation.

Reply:
Thanks and comments are added.